src/arrayMethodSort.js

```diff
@@ -4,8 +4,31 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (String(a) < String(b)) {
+        return -1;
+      }
+
+      if (String(a) > String(b)) {
+        return 1;
+      }
+
+      return 0;
+    },
+  ) {
+    for (let i = 0; i < this.length - 1; i++) {
+      for (let j = 0; j < this.length - 1 - i; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
+          const temp = this[j];
+
+          this[j] = this[j + 1];
+          this[j + 1] = temp;
```

Comment:
The swapping of elements is implemented correctly. As an alternative, you could use array destructuring to swap values: [this[j], this[j + 1]] = [this[j + 1], this[j]]. This can make the code a bit cleaner.